lib/ui/components/album_list_item_widget.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:dotted_border/dotted_border.dart';
+import 'package:flutter/material.dart';
+import 'package:logging/logging.dart';
+import 'package:photos/db/files_db.dart';
+import 'package:photos/models/collection_items.dart';
+import 'package:photos/theme/ente_theme.dart';
+import 'package:photos/ui/viewer/file/no_thumbnail_widget.dart';
+import 'package:photos/ui/viewer/file/thumbnail_widget.dart';
+
+class AlbumListItemWidget extends StatelessWidget {
+  final CollectionWithThumbnail? item;
+  final bool isNew;
```

Comment:
I am not sure if we should fix the Album with widget and isNew widget/button. Should we keep them separate? If it's too much overhead, can go with this.